android/libraries/rib-badoo/build.gradle

```diff
@@ -0,0 +1,56 @@
+configureAndroidLibrary(project)
+
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+
+android {
+    defaultConfig {
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+androidExtensions {
+    experimental = true
+}
+
+repositories {
+    maven { url 'https://jitpack.io' }
+}
+
+dependencies {
+    kapt deps.apt.raveCompiler
+    kapt deps.apt.daggerCompiler
+
+    implementation project(":libraries:rib-android")
+    implementation project(":libraries:rib-workflow")
+    implementation deps.external.kotlinStdlib
+    implementation deps.support.appCompat
+    implementation deps.support.archLifecycleCommon // TODO: remove
+    implementation "io.reactivex.rxjava2:rxkotlin:2.2.0" // TODO: remove
+    implementation deps.external.dagger
+    implementation deps.external.rxbinding
+    implementation deps.uber.rave
+    implementation deps.external.mviCore
```

Comment:
Is it ok to depend on MviCore? It depends on RxJava.

Reply:
I guess yes.